<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainModule</class>
 <widget class="QDialog" name="TrainModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TrainModule</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>动作时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_Do_Duration"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbUseRest">
      <property name="text">
       <string>使用休息态作为间隔：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_Rest_Duration"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>手势动作</string>
   </property>
   <widget class="QCheckBox" name="cbThumb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弯曲拇指</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbIndex">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弯曲食指</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMiddle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弯曲中指</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbRing">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弯曲无名指</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbLittle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弯曲小指</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEFinger">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Add_Finger">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>动作名称：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>手腕动作</string>
   </property>
   <widget class="QPushButton" name="Btn_Add_Wrist">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rBtn_Shangqie">
       <property name="text">
        <string>上切</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rBtn_Neifan">
       <property name="text">
        <string>内翻</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="rBtn_Neixuan">
       <property name="text">
        <string>内旋</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rBtn_Xiaqie">
       <property name="text">
        <string>下切</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="rBtn_Waifan">
       <property name="text">
        <string>外翻</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="rBtn_Waixuan">
       <property name="text">
        <string>外旋</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>手肘动作</string>
   </property>
   <widget class="QPushButton" name="Btn_Add_Elbow">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rBtn_Shenzhou">
       <property name="text">
        <string>伸肘</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rBtn_Quzhou">
       <property name="text">
        <string>屈肘</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="Tb_series">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>211</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>训练序列文件操作</string>
   </property>
   <widget class="QPushButton" name="Btn_Save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存当前序列...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Import">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入原有序列...</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Start">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始训练</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="Btn_Delete_Latest">
      <property name="text">
       <string>删除最新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Clear">
      <property name="text">
       <string>清除全部</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>肌电臂带连接设置</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,2,1,2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>串口号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbBox_Armband">
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_Armband">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>连接状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_Armband">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btn_DataSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数据保存...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
